airflow/dags/scrape_feed_aggregators/METADATA.yml

```diff
@@ -1,4 +1,4 @@
-description: "Check common GTFS feed aggregators for canonical URLs in agencies.yml"
+description: "Scrape GTFS (and other) feed URLs present in validators"
```

Comment:
Besides this, do we have any replacement for the deleted README that was in `jobs/gtfs-aggregator-checker`? Should we replace that README 1:1 with a new version in `jobs/gtfs-aggregator-scraper`

Reply:
Eh not really, I don't think a readme is that useful for this